src/main/java/com/jjoe64/graphview/Viewport.java

```diff
@@ -424,6 +431,11 @@ public boolean onScroll(MotionEvent e1, MotionEvent e2, float distanceX, float d
             {
                 return false;
             }
+            // The first scroll event should be ignored because of bad distanceX
```

Comment:
What is the value of distanceX when it is bad during the first scroll event?

Reply:
@mbalasciaps It is a known issue in GuestureDetector.SimpleOnGestureListener
Please look at the below link. I faced the same problem.
https://stackoverflow.com/questions/23824123/android-onscroll-event-giving-bad-values

Reply:
Thanks.